### DISCUSSION TITLE
Hacktoberfest 2022

### DISCUSSION THREAD

**Discussion by WillDaSilva** [meltano/meltano#6687] [General]
How should we handle [Hacktoberfest 2022](https://hacktoberfest.digitalocean.com/)? It's a month-long event ran by Digital Ocean wherein people are incentivized to create & merge PRs in public GitHub repositories. It has traditionally attracted a large amount of low-quality contributions.

Because of the low-quality contribution problem, Digital Ocean has made the event opt-in - public repositories need the "Hacktoberfest" topic to be eligable, or the issues/PRs need to have the "Hacktoberfest" label.

I propose that we *not* add the "Hacktoberfest" topic to our repositories or the "Hacktoberfest" label to our issues, but we can add the "Hacktoberfest" label to PRs after they've been made if they're of sufficiently high quality. This would avoid attracting low-quality contributions, but still reward those who are participating in Hacktoberfest and providing high-quality contributions.

We'll want to ensure that taptoberfest is not conflated with Hacktoberfest. Note that taptoberfest contributions could be used to make progress towards rewards from Hacktoberfest.

CC @tayloramurphy @aaronsteers

**Comment by tayloramurphy** [comment]
@WillDaSilva I think your proposal makes sense to me. We're not looking for just any random contribution. FYI @afolson since this is a consideration!
